README.md

```diff
@@ -1,23 +1,24 @@
-# Accurate Instagram Filters Reconstruction
+# Accurate Image Filters Reconstruction
 
 There's a bunch of apps out there (e.g., Instagram) allowing you to apply color filters to images.
 You might be interested in cloning their behavior: reconstruct them. This
 repository holds tools for semi-automatically and very accurately reconstructing
 color filters.
 
-The truth is folks like Instagram filters. They are trying to
+The truth is folks like filters. They are trying to
```

Comment:
Again, this is more about Instagram. Also, all examples are about Instagram filters.

Reply:
So, are we putting Instagram there or trying to find a way around?